src/store/state.js

```diff
@@ -9,9 +9,14 @@ const network =
     : nodeList['ETH'][0];
 const notifications =
   store.get('notifications') !== undefined ? store.get('notifications') : {};
+const regularGasPrice = network.type.regularGasPrice;
 const gasPrice =
-  store.get('gasPrice') !== undefined ? store.get('gasPrice') : 10;
-
+  store.get('gasPrice') !== undefined
+    ? store.get('gasPrice')
+    : regularGasPrice !== undefined
+    ? regularGasPrice
+    : 41;
+const minimumGasPrice = network.type.minimumGasPrice || 0;
```

Comment:
Let's try not to add a new property to the store/state. (see comment in actions.js)